rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -575,8 +575,8 @@ upgrade_slapdconf() {
   if ${NEED_REINDEX}; then
     # stop slapd, slapindex -q, start slapd. 
     systemctl stop rudder-slapd
-    echo -n "Reindexing LDAP directory"
-    /opt/rudder/sbin/slapindex -q
+    echo -n "Reindexing LDAP directory - this may take a few minutes..."
+    /opt/rudder/sbin/slapindex
```

Comment:
This needs to be run with the `rudder-slapd` user, cf. https://linux.die.net/man/8/slapindex